src/maintagent/agent/maintagent.c

```diff
@@ -56,6 +57,8 @@ int main(int argc, char **argv)
   COMMIT_HASH = fo_sysconfig("maintagent", "COMMIT_HASH");
   VERSION = fo_sysconfig("maintagent", "VERSION");
   snprintf(agent_rev, sizeof(agent_rev), "%s.%s", VERSION, COMMIT_HASH);
+  /* Get the Agent Key from the DB */
```

Comment:
This comment does not match. It should tell something like "update `agent_desc` if necessary"